tornado/ioloop.py

```diff
@@ -956,6 +966,7 @@ def start(self):
 
         finally:
             # reset the stopped flag so another start/stop pair can be issued
+            self._running = False
```

Comment:
This was not necessary for this PR, but it seems more robust to add this (if `loop.start` gets crashed by an exception, the "running" flag should be reset to False).